src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(val) {
+      this.result += val;
+
+      return this;
+    },
+    subtract(val) {
+      this.result -= val;
+
+      return this;
+    },
+    multiply(val) {
+      this.result *= val;
+
+      return this;
+    },
+    divide(val) {
+      if (val === 0) {
+        return 'Cannot divide by zero.';
```

Comment:
The `divide` method returns a string when dividing by zero, which breaks the method chaining pattern used in other methods. Consider throwing an error or handling this case differently to maintain consistency with the other methods.